src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -69,6 +69,7 @@ private static ModElement me(Workspace workspace, ModElementType type, String su
 			generatableElements.add(getRecipeExample(me(workspace, type, "5"), "Smoking", random, true));
 			generatableElements.add(getRecipeExample(me(workspace, type, "6"), "Stone cutting", random, true));
 			generatableElements.add(getRecipeExample(me(workspace, type, "7"), "Campfire cooking", random, true));
+			generatableElements.add(getRecipeExample(me(workspace, type, "8"), "Smithing", random, true));

```

Comment:
You will need to specify it too, just requesting example without defining it won't work.